tachyon/zk/plonk/permutation/column_store.h

```diff
@@ -0,0 +1,112 @@
+#ifndef TACHYON_ZK_PLONK_PERMUTATION_COLUMN_STORE_H_
+#define TACHYON_ZK_PLONK_PERMUTATION_COLUMN_STORE_H_
+
+#include <vector>
+
+#include "absl/types/span.h"
+#include "gtest/gtest_prod.h"
+
+#include "tachyon/zk/plonk/circuit/column.h"
+#include "tachyon/zk/plonk/circuit/columns.h"
+#include "tachyon/zk/plonk/permutation/lookup_table.h"
+
+namespace tachyon::zk {
+
+// |ColumnStore| has three tables with the same number of columns and rows
+// exist. In reality, for optimization purposes, it only has pointers to
+// tables. And |ColumnStore| supports functions that can query the columns of
+// each table coressponding specific chunk.
+// - |original|: 2-dimensinal vector with δ and w labels.
+// - |permuted|: permuted |original|
+// - |columns|: witness columns.
```

Comment:
Please update the comments.

Reply:
I think it is resolved. Plz close it @chokobole